ng2-components/ng2-alfresco-upload/src/components/upload-button.component.ts

```diff
@@ -80,6 +80,9 @@ export class UploadButtonComponent implements OnInit, OnChanges {
     currentFolderPath: string = '/';
 
     @Input()
+    uploadToECM: boolean = true;
```

Comment:
Please use the upload directive (it is inside the alfresco core component) instead of change the alfresco upload.